pkg/controller/infrastructure/infraflow/ensure_utils.go

```diff
@@ -20,6 +20,8 @@ const (
 	DefaultFlowSampling = 0.5
 	// DefaultMetadata is the default value for the Flow Logs metadata.
 	DefaultMetadata = "EXCLUDE_ALL_METADATA"
+	// DefaultSecondarySubnetName is the default name of the secondary ipv4 subnet that will be used in dualstack shoot
+	DefaultSecondarySubnetName = "ipv4-pod-cidr"
```

Comment:
please use this constant also in here:
```
pkg/controller/infrastructure/infraflow/ensure_utils.go
130:                            RangeName:   "ipv4-pod-cidr",
```

Reply:
And probably we should use that constant in conjunction with the cluster name. Because when we'd have another ipv6 cluster in the same VPC that will be a collision.